velox/docs/functions/spark/datetime.rst

```diff
@@ -34,6 +34,13 @@ These functions support TIMESTAMP and DATE input types.
     deducted from ``start_date``.
     Supported types for ``num_days`` are: TINYINT, SMALLINT, INTEGER.
 
+.. spark:function:: date_format(timestamp, dateFormat) -> string
+
+    Converts `timestamp` to a string in the format specified by `dateFormat`.
+
+        SELECT date_format('2020-01-29', 'yyyy'); -- '2020'
+        SELECT date_format('2024-05-30 08:00:00', 'yyyy-MM-dd'); -- '2024-05-30'
```

Comment:
I wonder if the patterns of 'dateFormat' are the same between Presto and Spark. If no, we might need separate implementation.

Reply:
Yes, JodaDateTimeFormatter is used as expected. And it has been validate by Spark UTs.